specs/sb/api.md

```diff
@@ -342,6 +390,29 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
```

Comment:
Good catch, thanks, I've updated for both service_instances and service_bindings.